components/browser/state/src/main/java/mozilla/components/browser/state/state/ContainerState.kt

```diff
@@ -2,7 +2,22 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-package mozilla.components.feature.containers
+package mozilla.components.browser.state.state
+
+/**
+ * Value type that represents the state of a container.
+ *
+ * @property contextId The session context ID also known as cookie store ID for the container.
+ * @property name Name of the container.
+ * @property color The color for the container. This can be shown in tabs belonging to this container.
+ * @property icon The name of an icon for the container.
+ */
+data class ContainerState(
+    override val contextId: String,
+    override val name: String,
+    override val color: String,
```

Comment:
I wonder why `Color` is a `String`? Wouldn't it make more sense to have an `Int` here that we can use as a color in UI code?

Reply:
Changed to `@ColorInt val color: Int,` in my next revision.